salt/states/user.py

```diff
@@ -359,6 +364,17 @@ def present(name,
 
         .. versionchanged:: 2015.8.0
     '''
+
+    '''
```

Comment:
Thanks for looking into this Mike. Just submitted the changes. 

Date: Fri, 12 Feb 2016 09:11:34 -0800
From: notifications@github.com
To: salt@noreply.github.com
CC: papoyan@msn.com
Subject: Re: [salt] added a feature to hash a password if provided as clear text (#31161)

In salt/states/user.py:

> @@ -359,6 +364,17 @@ def present(name,
>  
>          .. versionchanged:: 2015.8.0
>      '''
> +
> +    '''

Could you please prefix this comment with # symbols instead of using triple-quotes? This confuses our docs parser.


—
Reply to this email directly or view it on GitHub.